<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppLockerClass</class>
 <widget class="QMainWindow" name="AppLockerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppLocker - 如果不会用就点击&quot;帮助&quot;</string>
  </property>
  <property name="windowIcon">
   <iconset resource="applocker.qrc">
    <normaloff>:/png/res/locker.png</normaloff>:/png/res/locker.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>输入路径: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edit_in_path">
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_open_in_path">
        <property name="text">
         <string>打开...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>输出路径: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edit_out_path">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_open_out_path">
        <property name="text">
         <string>打开...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>加锁框参数(短编辑框最多输入15字节, 长编辑框最多输入63字节)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>窗体标题:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edit_mainwin_title">
         <property name="text">
          <string>LX牌登录系统</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>窗体创建时X坐标: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_mainX">
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>620</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>窗体创建时Y坐标: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spinBox_mainY">
         <property name="suffix">
          <string/>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>380</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>普通编辑框的名称: </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edit_fmtedit_name">
         <property name="text">
          <string>用户名:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>密码编辑框的名称: </string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="edit_pwdedit_name">
         <property name="text">
          <string>密码: </string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&quot;确认&quot;按钮的名称: </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="edit_ok_btn_name">
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&quot;退出&quot;按钮的名称: </string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="edit_exit_btn_name">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>普通编辑框的答案: </string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QLineEdit" name="edit_fmtedit_answer">
         <property name="text">
          <string>admin</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>密码编辑框的答案: </string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="3">
        <widget class="QLineEdit" name="edit_pwdedit_answer">
         <property name="text">
          <string>123456</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&quot;输入正确&quot;提示框的标题:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="3">
        <widget class="QLineEdit" name="edit_scsdlg_title">
         <property name="text">
          <string>输入正确</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>&quot;输入错误&quot;提示框的标题:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="3">
        <widget class="QLineEdit" name="edit_faliddlg_title">
         <property name="text">
          <string>输入错误</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>&quot;输入正确&quot;提示框的内容:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="3">
        <widget class="QLineEdit" name="edit_scsdlg_text">
         <property name="text">
          <string>输入正确, 您可以使用该软件了!</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&quot;输入错误&quot;提示框的内容:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="3">
        <widget class="QLineEdit" name="edit_faliddlg_text">
         <property name="text">
          <string>输入错误, 请重新输入!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_prev_dlg">
        <property name="text">
         <string>预览加锁框</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_begin_lock">
        <property name="text">
         <string>加锁!</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="act_useTip"/>
    <addaction name="act_warning"/>
    <addaction name="act_about"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="act_unlock"/>
    <addaction name="act_restore"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="act_unlock">
   <property name="text">
    <string>去除加锁框</string>
   </property>
  </action>
  <action name="act_useTip">
   <property name="text">
    <string>使用方法</string>
   </property>
  </action>
  <action name="act_about">
   <property name="text">
    <string>关于 AppLocker...</string>
   </property>
  </action>
  <action name="act_virus">
   <property name="text">
    <string>病毒制作</string>
   </property>
  </action>
  <action name="act_warning">
   <property name="text">
    <string>注意事项</string>
   </property>
  </action>
  <action name="act_restore">
   <property name="text">
    <string>恢复默认设置</string>
   </property>
  </action>
  <action name="act_checkUpdate">
   <property name="text">
    <string>检查更新</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="applocker.qrc"/>
 </resources>
 <connections/>
</ui>
